src/main/java/dev/jbang/cli/Run.java

```diff
@@ -80,7 +83,7 @@ public Integer doCall() throws IOException {
 	}
 
 	RunContext getRunContext() {
-		RunContext ctx = RunContext.create(userParams, null,
```

Comment:
observation: this line is going to grow as we  add all of the options here :)

Reply:
Yeah, we'll see how long it gets before we get sick of it and refactor the shit out of that :-)